### ISSUE TITLE
why I can't load the pretrained model 

### ISSUE THREAD

**Issue by rain2008204** [Joker316701882/Additive-Margin-Softmax#12]
the error is Key InceptionResnetV1/Block8/Branch_0/Conv2d_1x1/BatchNorm/beta/Adam not found in checkpoint
@

**Comment by rain2008204** [comment]
@Joker316701882
